### PR TITLE
Fix issue1597

### PR DESCRIPTION
**Pull request by youngsterxyf**
R.T. #1597

### CODE REVIEW

**Comment by astaxie** [comment]
@youngsterxyf build failed

**Comment by youngsterxyf** [comment]
@astaxie maybe this failure is not related with my pr，but i will continue to check the reason。

image

**Comment by youngsterxyf** [comment]
@astaxie I made no change, but it succeed. :smile:

**Comment by astaxie** [comment]
@youngsterxyf weird. I will review later.

**Comment by youngsterxyf** [comment]
@astaxie It's a bug of go or travisCI ?

**Comment by astaxie** [comment]
it looks like the timeout and autoclose the connection. then call the WaitGroup.Done(). I see the same issue in beego's grace module. I still have no idea on that. but we can recover from the panic

File: beego.go [hunk 1]
```diff
@@ -73,13 +72,6 @@ func initBeforeHTTPRun() {
 	if err != nil {
 		panic(err)
 	}
-	//init log
-	for adaptor, config := range BConfig.Log.Outputs {
-		err = BeeLogger.SetLogger(adaptor, config)
-		if err != nil {
-			fmt.Printf("%s with the config `%s` got err:%s\n", adaptor, config, err)
-		}
-	}
```
Comment (astaxie) [inline beego.go]: why I add these init log here. As user can change the config path. So if they change that, we need to init the BeeLogger again.

**Pull request closed without merging**